leapp/cli/upgrade/__init__.py

```diff
@@ -67,6 +67,12 @@ def upgrade(args):
             raise CommandError('No previous upgrade run to continue, remove `--resume` from leapp invocation to'
                                'start a new upgrade flow')
         os.environ['LEAPP_DEBUG'] = '1' if os.getenv('LEAPP_DEBUG', configuration.get('debug', '0')) == '1' else '0'
+
+        if os.environ['LEAPP_DEBUG'] == '1' or os.getenv('LEAPP_VERBOSE', configuration.get('verbose', '0')) == '1':
```

Comment:
If you want to make use of configuration.get('verbose', '0'), you will also have to set it above. See line 61 here.